appveyor.yml

```diff
@@ -0,0 +1,115 @@
+pull_requests:
```

Comment:
Could you rename the appveyor.yml file into a dot-file-style YAML :point_right:  `.appveyor.yml`?